src/traits/trait-objects.md

```diff
@@ -0,0 +1,87 @@
+# Trait Objects
+
+We've seen how a function can take arguments which implement a trait:
```

Comment:
I think this sentence needs to be updated?

Reply:
Good catch. Reworded it.